### ISSUE TITLE
Changing repos from http to https causes error [Tigris #912]

### ISSUE THREAD

**Issue by markphip** [subclipse/archive#154]
## Description

```
1. I have a project checked out from repository http://xxx/.

2. I deleted the project from Eclipse (and told it to delete from the filesystem too).

3. I added a new repository, https://xxx/, and deleted the http one.

4. I checked out the project again. The checkout seemed to work, but then I got a bunch of errors in 
the console. It looks like something is still remembering the old http URL. The errors look like this:
    RA layer request failed
svn: OPTIONS of 'http://xxx/yyy': Could not read status line: connection was closed by server 
(http://xxx)

5. I tried deleting and checking out the project again -- same thing.

6. Only after restarting Eclipse was I able to check it out without errors.


I'm using SVN 1.6.1 with Subclipse 1.6.2 and JavaHL 1.6.1.
```
## Metadata Imported from Tigris (Issue 912)
- **Creation Date**: 2009-05-05 15:33:35
- **Reporter**: jacobweber
- **Subcomponent**: core
- **Version**: current
- **Milestone**: not determined
- **Keywords**: 
- **Cc**: 
### 2009-05-06 16:56:57 by markphip

```
*** Issue 913 has been marked as a duplicate of this issue. ***
```
### 2009-05-06 16:57:44 by markphip

```
Can you make sure you are using JavaHL and not SVNKit?  I do not believe this
could be a problem in Subclipse.  Maybe if you are using SVNKit it has some bug
where it has hung on to a connection.
```
### 2009-05-06 18:27:43 by jacobweber

```
Yep, JavaHL (JNI) 1.6.1 (r37116).
```
